x/paramfilter/test/param_filter_test.go

```diff
@@ -34,30 +33,6 @@ func TestParamFilter(t *testing.T) {
 		require.Error(t, err)
 		require.Contains(t, err.Error(), "parameter can not be modified")
 	}
-
-	// ensure that we are not filtering out valid proposals
-	validChange := proposal.NewParamChange(stakingtypes.ModuleName, string(stakingtypes.KeyMaxValidators), "1")
-	p := testProposal(validChange)
-	err := handler(ctx, p)
-	require.NoError(t, err)
-
-	ps := app.StakingKeeper.GetParams(ctx)
-	require.Equal(t, ps.MaxValidators, uint32(1))
-
-	// ensure that we're throwing out entire proposals if any of the changes are blocked
-	for _, p := range app.BlockedParams() {
-		// try to set the max validators to 2, unlike above this should fail
-		validChange := proposal.NewParamChange(stakingtypes.ModuleName, string(stakingtypes.KeyMaxValidators), "2")
```

Comment:
[blocking] good catch! IMO we should update this test to use a governance modifiable param instead of deleting it entirely.